js/translation-helpers.js

```diff
@@ -71,8 +81,13 @@ $gp.translation_helpers = (
 				tab_select: function() {
 					$gp.translation_helpers.tab_select( $( this ) );
 					return false;
-				}
-			}
+				},
+                reply_comment_form: function ( event ) {
+                    event.preventDefault();
+					$gp.translation_helpers.reply_comment_form( $( this ) );
+                    return false;
+               },
```

Comment:
Mix of tabs and spaces:
```suggestion
				reply_comment_form: function ( event ) {
					event.preventDefault();
					$gp.translation_helpers.reply_comment_form( $( this ) );
					return false;
				},
```